wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+  <v-progress-linear v-if="isLoading" color="primary" indeterminate />
+  <v-card class="ma-10">
+    <v-row align="center">
+      <v-col cols="6" align="center">
+        <v-tooltip
+          text="You must be Master of the Universe and 21 or older to add words"
+          :disabled="isOlderThanTwentyOne && isMotU">
+          <template v-slot:activator="{ props }">
+            <v-btn
+              v-bind="props"
+              :variant="isOlderThanTwentyOne && isMotU ? 'elevated' : 'plain'"
+              :color="isOlderThanTwentyOne && isMotU ? 'primary' : ''"
+              class="ma-0"
+              @click="if (isOlderThanTwentyOne && isMotU) addWordDialog = true;"

```

Comment:
```suggestion
        :disabled="!(isOlderThanTwentyOne && isMotU)"
              @click="addWordDialog = true"
```

You can utilize the disabled field on v-btn to make so the user can't click if they arne't MOTU and under 21. Right now, it shows the tool tip but I can still click it and get feedback

Reply:
That's a problem I had with the tooltip. I talked with Meg about it and we weren't able to figure out how to show the tooltip on disabled with a good placement of the tooltip.